articles/quickstart/native/android/_includes/_login.md

```diff
@@ -65,6 +65,8 @@ You do not need to declare a specific `intent-filter` for your activity, because
 The `AndroidManifest.xml` file should look like this:
 
 ```xml
+// app/src/main/AndroidManifest.xml
```

Comment:
I don't think `//` comment syntax is valid for XML.
AFAIK, the only way to add a comment in XML is the following:

```
<!-- to be commented xml block goes here -->
```

Can you please change this code block to use the proper syntax or suggest a working alternative?
Thanks!

Reply:
I was thinking the same, then I saw some comments in same block using `//` so used it. Updating it now